pkgs/development/python-modules/py-opensonic/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "py-opensonic";
+  version = "5.0.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "khers";
+    repo = "py-opensonic";
+    rev = "v${version}";
+    hash = "sha256-FZJ8njXdSaNuwX9pmFV8Gb7BpTS7Y09h9ZNSjcXkkc8=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
```

Comment:
Why do we need wheel here? The package is not referring to it except through GHA.